package.json

```diff
@@ -9,16 +9,26 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@mdx-js/loader": "^2.1.5",
+    "@mdx-js/react": "^2.1.5",
+    "@next/mdx": "^13.0.4",
+    "@types/mdx": "^2.0.3",
     "babel-plugin-styled-components": "^2.0.7",
     "bootstrap": "^5.2.2",
     "gray-matter": "^4.0.3",
     "marked": "^4.1.1",
     "next": "13.0.0",
+    "next-mdx-remote": "^4.2.0",
     "react": "18.2.0",
     "react-bootstrap": "^2.5.0",
     "react-dom": "18.2.0",
     "react-multi-carousel": "^2.8.2",
     "react-responsive-carousel": "^3.2.23",
+    "react-syntax-highlighter": "^15.5.0",
+    "rehype-highlight": "^6.0.0",
+    "rehype-katex": "^6.0.2",
+    "remark-frontmatter": "^4.0.1",
+    "remark-math": "^5.1.1",
```

Comment:
You're not using any of these directly, are you sure you need these dependencies?

Reply:
No, you are right.
I'm doing some experiments for the LaTeX rendering and I'm using more deps than I need.
After the final solution will be found, I planned to make a cleaning removing the useless modules from the `yarn` deps.